locator/tablets.cc

```diff
@@ -220,6 +223,23 @@ dht::token_range tablet_map::get_token_range(tablet_id id) const {
     }
 }
 
+seastar::future<std::vector<token>> tablet_map::get_flat_token_ranges() const {
+    auto tokens = make_lw_shared<std::vector<token>>();
+    return for_each_tablet([this, tokens](locator::tablet_id id, const locator::tablet_info& info) -> future<> {
```

Comment:
isn't it better to simplify it with coroutines?

Reply:
Fixed - thanks! New patch will be uploaded soon.